### DISCUSSION TITLE
Welcome to DB48X Discussions!

### DISCUSSION THREAD

**Discussion by c3d** [c3d/db48x#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by balta1980** [comment]
Hi thank you for all your efforts and work. I appreciate that. it is not too abusive would you be able to provide the firmware? So the people don't need to develop for them selves? Thank you.

**Reply by balta1980** [reply]
I am sorry I mean build instead of develop.

**Reply by c3d** [reply]
This is planned for "very soon now™"

I want the firmware to be somewhat usable and stable before cutting a 0.1 release. A lot of progress towards that goal was made just this week-end, notably being able to deliver a QSPI image that lets you run the DM42 program as well.

**Reply by c3d** [reply]
Hey @balta1980, I just build v0.1 now. Check it out and tell me if it works for you.

**Reply by balta1980** [reply]
Thank you I will check it ASAP and revert to you.

**Comment by balta1980** [comment]
I on it in my calc thank you
